src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -3,10 +3,15 @@
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.ResultSet;
 
 public class DatabaseManager {
     private static final String PORT = "5432";
     private static final String SERVER = "localhost";
+    private static final String TABLE_NAMES_QUERY = "SELECT relname FROM pg_stat_user_tables ORDER BY relname";
```

Comment:
Why you made this query as a constant? You're using it just once. It means that it could be a local variable

Reply:
Could be a local variable

Reply:
This is one of the programming styles.
For me it's an experiment, I want to try to make all the numbers into constants.